common/src/main/webapp/sass/visualization.scss

```diff
@@ -138,12 +138,14 @@ $brightness: var(--aladin-image-brightness);
 
   .circle-target {
     display: none;
+    pointer-events: fill;
```

Comment:
What is the difference in this case between the default (`auto`/`visiblePainted`) and `fill`?

Reply:
I tried several combinations that would work on FF and Chrome and `auto/visiblePainted` didn't. Couldn't find any documentation that would indicate why they are different